src/js/components/agency/statusOfFunds/VisualizationSection.jsx

```diff
@@ -52,22 +57,132 @@ const VisualizationSection = ({
+    }, [isMobile, viewType, windowWidth]);
+
+    const columns = toggle ?
+        [
+            {
+                title: 'subComponent',
+                displayName: 'Sub-Component'
+            },
+            {
+                title: 'outlays',
+                displayName: [`${fyString} Outlays`]
+            }
+        ]
+        :
+        [
+            {
+                title: 'subComponent',
+                displayName: 'Sub-Component'
+            },
+            {
+                title: 'totalBudgetaryResources',
+                displayName: isMobile ? 'Total Budgetary Resources' : [`${fyString} Total Budgetary`, <br />, 'Resources']
+            },
+            {
+                title: 'obligations',
+                displayName: `${fyString} Obligations`
+            }
+        ];
+
+    const rows = results.map((data) => (toggle ?
+        [
+            (
+                <div>
+                    {data.name}
+                </div>
+            ),
+            (
+                <div>
+                    {formatMoneyWithPrecision(data._outlays)}
+                </div>
+            )
+        ]
+        :
+        [
+            (
+                <div>
+                    {data.name}
+                </div>
+            ),
+            (
+                <div>
+                    {formatMoneyWithPrecision(data._budgetaryResources)}
+                </div>
+            ),
+            (
+                <div>
+                    {formatMoneyWithPrecision(data._obligations)}
+                </div>
+            )
+        ]));
+
+    const changeView = (label) => {
+        setViewType(label);
+    };
+
+    const chartTableToggle = (
+        <ChartTableToggle
+            activeType={viewType}
+            changeView={changeView} />
+    );
+
     return (
         <div
             className="status-of-funds__visualization"
             onMouseLeave={() => {
                 const el = document.querySelector("div.tooltip-wrapper.sof_chart-tt");
                 el.style.display = "none";
             }}>
-            <h6>{level === 1 ? selectedSubcomponent?.name : agencyName} by <span className="status-of-funds__emphasis">{levels[level]}</span> for FY {fy}</h6>
-            <div
-                className="status-of-funds__controls">
-                <RoundedToggle toggle={toggle} onKeyToggle={onKeyToggle} onToggle={onToggle} label="View Outlays" />
-                <div className="status-of-funds__line-div" />
-                <div className="status-of-funds__accordion">
-                    <Accordion setOpen={setOpen} closedIcon="chevron-down" openIcon="chevron-up" title="What is this?" />
-                </div>
-            </div>
+            {isMobile ? (
+                <>
+                    <h6>{level === 1 ? selectedSubcomponent?.name : agencyName} by <span className="status-of-funds__emphasis">{levels[level]}</span> for FY {fy}
```

Comment:
should you be using a different heading tag and then just making the font size the size of the h6 tag?